dev-libs/yazpp/yazpp-1.8.1.recipe

```diff
@@ -0,0 +1,159 @@
+DESCRIPTION="YAZ++ is a C++ toolkit for development of Z39.50 and SRU clients and servers. \
+It is built on top of the YAZ toolkit and provides additional C++ classes and utilities."
+HOMEPAGE="https://software.indexdata.com/yazpp/doc/"
+COPYRIGHT="1995-2023 Index Data"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+portVersion="1.8.1"
+SOURCE_URI="https://ftp.indexdata.com/pub/yazpp/yazpp-$portVersion.tar.gz"
+CHECKSUM_SHA256="95a58d9c7cfd640572b1bbeb5849097b0b9c02e8b3b8eb2985b13b87dd07eb6a"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="6.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%.*}"
+
+PROVIDES="
+	yazpp$secondaryArchSuffix = $portVersionCompat
+	lib:libyazpp$secondaryArchSuffix = $libVersionCompat
+	lib:libzoompp$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libyaz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	yazpp${secondaryArchSuffix}_devel = $portVersionCompat
+	cmd:yazpp_config$commandSuffix
+	devel:libyazpp$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	yazpp$secondaryArchSuffix == $portVersion base
+	devel:libyaz$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	yazpp${secondaryArchSuffix}_tools = $portVersionCompat
+	cmd:yazpp_illclient$commandSuffix
+	cmd:yazpp_ztest$commandSuffix
+	"
+
+REQUIRES_tools="
+	yazpp$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	lib:libyaz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libxslt$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libyaz$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libxslt$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage yazpp$secondaryArchSuffix \
+	$libDir/libyazpp.so.$libVersion
+
+PATCH()
+{
+	# Fix library paths for Haiku
+	sed -i "s|/lib\"|/$libDir\"|g" configure.ac
+}
+
+BUILD()
+{
+	export LDFLAGS="-lnetwork"
+	
+	# Create symlink for libxml headers that YAZ expects
+	mkdir -p /tmp/include
+	ln -sf $portPackageLinksDir/devel~libxml2$secondaryArchSuffix/develop/headers/libxml2/libxml /tmp/include/libxml
+	export CPPFLAGS="-I/tmp/include"
+	
+	autoreconf -fi
+	
+	runConfigure ./configure \
+		--prefix=$prefix \
+		--bindir=$commandBinDir \
+		--libdir=$libDir \
+		--includedir=$includeDir \
+		--datadir=$dataDir \
+		--mandir=$manDir \
```

Comment:
These paths are normally passed by runConfigure already (except --bindir), so they shouldn't be needed explicitly.